packages/eslint-plugin/tests/rules/sort-ngmodule-metadata-arrays/cases.ts

```diff
@@ -156,8 +157,8 @@ export const invalid = [
     annotatedOutput: `
     @NgModule({
       [\`bootstrap\`]: [
-        AppModule2,
         AppModule1,
+        AppModule2,
```

Comment:
This change is due to the fact that the fixer now, fixes everything at once, not one-by-one as previously.